spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/support/DefaultSubscriberFactory.java

```diff
@@ -48,10 +51,13 @@
 
 	private final CredentialsProvider credentialsProvider;
 
+	private final Long pullSimpleTimeoutNoRetriesMillis;
+
 	public DefaultSubscriberFactory(GcpProjectIdProvider projectIdProvider,
 			ExecutorProvider executorProvider,
 			TransportChannelProvider channelProvider,
-			CredentialsProvider credentialsProvider) {
+			CredentialsProvider credentialsProvider,
+			Long pullSimpleTimeoutNoRetriesMillis) {
```

Comment:
In Spring we follow rule like: if options isn't mandatory, it has to be exposed via setter.
We had such a discussion in other PRs. Why are we here again?
Thanks

Reply:
Oops. :)